src/database/dbBuild.sql

```diff
@@ -0,0 +1,60 @@
+BEGIN;
+
+  DROP TABLE IF EXISTS users, categories, plans, category_plan, expenses;
+
+  CREATE TABLE users
```

Comment:
```suggestion
  CREATE TABLE users
```
Table names should preferably be a single name
but the name `user` the reserved word in sql
so you can to solve this write the table name in quotations

```SQL
 CREATE TABLE "user"
```